package.json

```diff
@@ -65,12 +66,12 @@
   "scripts": {
     "start": "node build/src/server.js",
     "dev": "NODE_ENV=development nodemon src/server.js --exec babel-node --inspect=0.0.0.0:9229 --nolazy",
-    "dev_docker": "NODE_ENV=development ./node_modules/.bin/nodemon src/server.js --exec babel-node --inspect=0.0.0.0:9229 --nolazy",
-    "debug": "node --inspect=0.0.0.0:9222 --nolazy build/src",
-    "build": "NODE_ENV=production gulp",
+    "dev_docker": "NODE_ENV=development ./node_modules/.bin/nodemon src/server.js --exec babel-node --inspect-brk=0.0.0.0:9229 --nolazy",
+    "debug": "babel-node --inspect-brk=0.0.0.0:9222 --nolazy build/src",
```

Comment:
should this one not be 9229?

Reply:
Fixed. Using same inspect port for docker and non-docket